docs/red_eyes_mainnet_tutorial.rst

```diff
@@ -0,0 +1,142 @@
+Raiden Red Eyes Mainnet Tutorial
+########################################
+
+.. toctree::
+  :maxdepth: 3
+
+Introduction
+=============
+In this tutorial we show how to use Raiden to do off chain payments using the Raiden Network on the Ethereum mainnet. For this tutorial we use the Red Eyes (LINK TO RELEASE) release. More information on the Red Eyes release can be found here(INSERT LINK ONCE READY). Since the Red Eyes release is a bug bounty release(LINK TO BLOGPOST), certain limits have been made to the amount of tokens that can be deposited in channels. This is done in order to minimize the funds that are potentially lost in case something goes wrong.
+
+Raiden has a Restful API with URL endpoints corresponding to actions that users can perform with their channels. The endpoints accept and return JSON encoded objects. The API URL path always contains the API version in order to differentiate queries to different API versions. All queries start with: ``/api/<version>/`` where ``<version>`` is an integer representing the current API version. (TODO: this part might be superfluous)
+
+Before getting started with this tutorial, please see the :doc:`Installation Guide <overview_and_guide>`, to make sure that Raiden is correctly installed and running.
+
+.. _join-token-network:
+
+Joining a token network
+==============================
+The first thing we need to do is to join a token network. In this case we want to join the Raiden token (`RDN <https://etherscan.io/token/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6>`_) network.
+
+*Note*: 1 RDN == 2**18 Rei. For the sake of readability and simplicity all token values in this tutorial are denominated in Rei and not RDN.
+
+In order to do so, we need the address of the token and the initial amount of tokens that we want to join the network with::
+
+    PUT /api/v2/connections/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6
+
+TODO: check correctness of API
+With the payload::
+
+    {
+        "funds": 20000
+    }
+
+This automatically connects our node to 5(TODO) other nodes in the network with 20000 / 5 Rei tokens in each channel.
```

Comment:
Since `5(TODO)` is there, i'd like to point our that the old docs mention it's 3 
https://raiden-network.readthedocs.io/en/stable/api_walkthrough.html#connect
Did we increase this from 3 to 5?

Reply:
We did not. It's just my memory that serves me wrong.